docs/bucket/notifications/README.md

```diff
@@ -638,9 +649,13 @@ mysql> select * from minio_images;
 <a name="apache-kafka"></a>
 ## Publish Minio events via Kafka
 
-Install kafka from [here](http://kafka.apache.org/).
+Install Apache Kafka from [here](http://kafka.apache.org/).
+
+### Step 1: Ensure minimum requirements are met
 
-### Step 1: Add kafka endpoint to Minio
+We tested Minio with Kafka version version 0.10.1.0. More recent versions should also work. Internally Minio uses the [Shopify/sarama](https://github.com/Shopify/sarama/) library distributed from [gopkg.in](http://gopkg.in/Shopify/sarama.v1), and any version supported by that library should work.
```

Comment:
Text here is a little vague, can you try to be specific about the supported version?